### PR TITLE
LPS-193178 Frontend tests

### PR DESCRIPTION
**Pull request by veroglez**
This PR depends on https://github.com/liferay-echo/liferay-portal/pull/13271

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-reports-web/test/js/components/render_times/getFragmentsByFilterValue.test.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		name: 'Grid',
+	},
+	{
+		cached: true,
+		fragment: false,
+		fromMaster: true,
+		name: 'Footer',
+	},
+];
+describe('convertRGBtoHex', () => {
```
Comment (victorg1991) [inline modules/apps/layout/layout-reports-web/test/js/components/render_times/getFragmentsByFilterValue.test.js:46]: :P

**Comment by victorg1991** [comment]
Just a small typo 😇

**Comment by veroglez** [comment]
> mes/getFragmentsByFilterValue.test.js



> Just a small typo 😇

Done!

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:stop

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Comment by ealonso** [comment]
ci:forward:force

**Comment by ealonso** [comment]
ci:forward:force

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/139659

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**